### ISSUE TITLE
review all pending/disabled tests

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#3048]
Review pending and disabled tests for release-1.5.0+ and go through all tests, remove tests that are disabled/pending that cannot be fixed  (this is why we have version control). 

- reaction test should run with no pending test.
- all tests that were disabled for release-1.4 and 1.5 should be fixed

**Comment by kieckhafer** [comment]
I don't see why we are expecting the last `expect` here to equal `isVisible`. We perform the same check three lines above this, and `expect` it to equal `!isVislble`, and we make no changes to anything that would constitute `isVisible` changing. Perhaps the last line of this test is wrong, and should be `!isVisible`?
```
it.skip("should let admin toggle product revision visibility", function () {
      sandbox.stub(Reaction, "hasPermission", () => true);
      const product = addProduct();
      let productRevision = Revisions.findOne({ documentId: product._id });
      const isVisible = productRevision.documentData.isVisible;
      expect(() => Meteor.call("products/publishProduct", product._id)).to.not.throw(Meteor.Error, /Access Denied/);
      productRevision = Revisions.findOne({ documentId: product._id });
      expect(productRevision.documentData.isVisible).to.equal(!isVisible);
      expect(() => Meteor.call("products/publishProduct", product._id)).to.not.throw(Meteor.Error, /Bad Request/);
      productRevision = Revisions.findOne({ documentId: product._id });
      expect(productRevision.documentData.isVisible).to.equal(isVisible);
    });
```

**Comment by mikemurray** [comment]
Some newlines between chunks of code would make this 1000% more readable :)

This code its attempt to test toggling the visibility, but I may be using the wrong method. Maybe it was meant to use `products/toggleVisibility`

**Comment by spencern** [comment]
Resolved by #3077
